util/fetch_iso639-1_codelist.py

```diff
@@ -0,0 +1,30 @@
+"""
+Updates the ISO639-1 codelist from the Library of Congress TSV file.
+"""
+
+import csv
+import os
+import requests
+
+from helper import schema_dir
+
+
+if __name__ == '__main__':
+
+    codes = {}
+
+    response = requests.get('http://id.loc.gov/vocabulary/iso639-1.tsv')
```

Comment:
```suggestion
    # https://www.iso.org/iso-639-language-codes.html links to loc.gov.
    response = requests.get('http://id.loc.gov/vocabulary/iso639-1.tsv')
```

Reply:
Done in https://github.com/open-contracting/standard/pull/1316/commits/aca29923c8749cd573dc0433d44adfeec9e84564